chat-plugins/mafia.js

```diff
@@ -1188,6 +1200,32 @@ exports.commands = {
 			`/mafia resethammer - sets the hammer to the default, resetting lynches`,
 		],
 
+		disablenl: 'enablenl',
+		enablenl: function (target, room, user, connection, cmd) {
+			let targetRoom = room;
+			target = target.split(',');
+			if (Rooms(target[0]) && Rooms(target[0]).users[user.userid]) targetRoom = Rooms(target.shift());
+			if (!targetRoom || !targetRoom.game || targetRoom.game.gameid !== 'mafia') return this.errorReply(`There is no game of mafia running in this room.`);
+			if (!user.can('mute', null, room) && targetRoom.game.hostid !== user.userid) return user.sendTo(targetRoom, `|error|/mafia ${cmd} - Access denied.`);
+			if (cmd === 'disablenl') {
+				if (!targetRoom.game.enableNL) return this.errorReply(`No-Lynch has already been disabled`);
```

Comment:
Don't user errorReply in mafia commands that can be called by buttons in the HTML room, the error goes to lobby.
Use ``user.sendTo(targetRoom, `|error|...`);``